srcpkgs/easyrpg-player/template

```diff
@@ -1,19 +1,16 @@
 # Template file for 'easyrpg-player'
 pkgname=easyrpg-player
-version=0.6.2.3
+version=0.8.1.1
 revision=1
-build_style=gnu-configure
-hostmakedepends="pkg-config autoconf automake libtool"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
```

Comment:
is this necessary? we set `CMAKE_BUILD_TYPE=None` to prevent some projects from doing weird things in release mode